test/unit/Reflection/ReflectionFunctionAbstractTest.php

```diff
@@ -349,6 +349,7 @@ public function returnTypeFunctionProvider()
             ['returnsString', 'string'],
             ['returnsNull', 'null'],
             ['returnsObject', '\stdClass'],
+            ['returnsVoid', 'void'],
```

Comment:
So I can now call `ReflectionFunctionAbstract#returnsVoid()`? Note that the method seems to be defined via magic, so some sort of documentation (even via docblock) would make things easier